src/app/company-page/dialog/remove-dialog/remove-dialog.component.css

```diff
@@ -0,0 +1,5 @@
+.button-click {
```

Comment:
Would `button-wrapper` be more suggestive? Or even `button-box`?

Reply:
I changed to `button-box-remove`